Shifty.Generated.ApiClient/OpenApiSpecs/AnalogCoreV2.json

```diff
@@ -17,7 +17,7 @@
   },
   "servers": [
     {
-      "url": "https://core.dev.analogio.dk"
+      "url": "http://localhost:5000"
```

Comment:
Can we make sure it uses the most recent version of dev and not whatever local branch you are on? 😅
It looks correct but just to make sure - assuming of course the backend part is finalized and merged?